pytensor/link/c/params_type.py

```diff
@@ -810,7 +810,7 @@ def c_support_code(self, **kwargs):
             struct_extract_method=struct_extract_method,
         )
 
-        return list(sorted(list(c_support_code_set))) + [final_struct_code]
+        return list(c_support_code_set) + [final_struct_code]
```

Comment:
I'm skeptical about some of these changes. Especially this one since it entirely drops a `sort`. Is this a bug in `flake8-comprehensions`?

Reply:
Could have been my mistake, let me check

Reply:
Did you see an issue with any other changes?